modules/fundamental/src/sbom/service/sbom.rs

```diff
@@ -460,7 +460,19 @@ impl SbomService {
             )
             .await?;
 
-        Ok(result.items.into_iter().map(|r| r.package).collect())
+        // TODO: this will break when adding pagination, as we effectively only process a single page
```

Comment:
can we leverage some `DISTINCT` on the query?

Reply:
Once we need this, we'd need to re-implement the query to return only packages, instead of full package-relationships. Currently, this re-uses the "fetch package relationships" function, stripping away the unnecessary information.